enarx-keep/tests/bin/write_emsgsize.c

```diff
@@ -0,0 +1,22 @@
+#include <sys/syscall.h>
+#include <unistd.h>
+#include <errno.h>
+
+void _start(void) {
+    char msg[128 * 1024];
```

Comment:
How is this size chosen for "too large"?

Reply:
The largest single write that I would consider "reasonable" is the ~64k maximum size of a UDP packet.